recipe/meta.yaml

```diff
@@ -573,6 +573,8 @@ requirements:
     - sysroot_linux-64 {{ cudavars[major_minor]["sysroot_version"] }}       # [linux64]
     - sysroot_linux-aarch64 {{ cudavars[major_minor]["sysroot_version"] }}  # [aarch64]
 {% endif %}
+  run:
+    - cuda-version {{ major_minor }}
```

Comment:
Contrary to its name, `major_minor` appears to contain update (patch) version info, too? It's the key used in the `cudavars` dict here: https://github.com/conda-forge/cudatoolkit-feedstock/blob/6b51fee9b74a9720623ccf5ebde48b06ed7c4bd7/recipe/meta.yaml#L468

and the default is `11.8.0`: https://github.com/conda-forge/cudatoolkit-feedstock/blob/6b51fee9b74a9720623ccf5ebde48b06ed7c4bd7/recipe/meta.yaml#L6